contracts/kleros/POC/KlerosPOCCourt.sol

```diff
@@ -1,5 +1,6 @@
 pragma solidity ^0.4.15;
 
+import "kleros-interaction/contracts/standard/arbitration/ArbitrableTransaction.sol";
```

Comment:
Why is this needed here?

Reply:
Oh, so it can be compiled, because it is from another repo?

Reply:
yes, it was exactly the reason. It seems that truffle wound't compile it if it is not referenced anywhere in the code.